README.md

```diff
@@ -5,8 +5,9 @@ This repository contains the source code for BinSkim, a Portable Executable (PE)
 ## For Developers
 
 1. Fork the repository -- **[Need Help?](https://help.github.com/articles/fork-a-repo/)**
```

Comment:
>Fork the repository [](http://example.com/codeflow?start=3&length=19)

One thing, when we encourage users to fork the repository, we lead them to submit PRs that can't be validated!

Are we set up to allow people without any explicit permissions in the repo to create a branch off and submit a PR?

I can ask someone on the team to run a test.

Reply:
Wait! Shaopeng, don't you have a different non-MS GitHub account? If so, can you try to enlist, then create a branch, then push that branch, then touch a file, and submit a PR?

The question is whether this works for you.

Reply:
behold :)
this is from a new account I just created and a touch of the file that is safe to merge in,
please click approve the workflow run and we can continue the flow to merge it in to verify this procedure works:
https://github.com/microsoft/binskim/pull/867

Reply:
oh I can approve, I have clicked that.

Reply:
I have left my comments in that PR. https://github.com/microsoft/binskim/pull/867